modules/querying/pages/func/mathematical-functions.adoc

```diff
@@ -477,17 +503,30 @@ Returns the square root of a number
 
 ==== Syntax
 
-`trunc( num, [decimal_place] )`
+`trunc( num, [decimal_places] )`
 
 
 ==== Description
 
-Returns a number truncated to a specified decimal place.
+Returns a number truncated to the specified number of decimal places.
+
+If the number of decimal places is either 0 or omitted, then the number is truncated to an integer value.
+The `decimal_places` parameter can be negative, meaning the target precision is to the left of the decimal point.
+This effectively rounds the number to some power of 10.
 
 
 ==== Return type
 
-A numeric type.
+A numeric type:
+
+* The same type as the input, if `decimal places` is positive.
```

Comment:
In our current implementation, our type checker records the `trunc` function as having type `double -> double` or `(double, int64) -> double`. The return type is technically double, but it will be compatible with any numeric type due to automatic type conversion. I'm not sure how much information we want to provide here.

Reply:
@DavidFan2000 Thanks. I'll just say the return type is double.